python/pyarrow/_substrait.pyx

```diff
@@ -26,6 +27,13 @@ from pyarrow.includes.libarrow cimport *
 from pyarrow.includes.libarrow_substrait cimport *
 
 
+def _check_conversion_options():
+    cdef:
+        CConversionOptions conv_opts
+
+    return conv_opts.strictness == BEST_EFFORT
+
+
```

Comment:
```suggestion
```

Let's get rid of this.  It'll just confuse future maintainers more than it helps I think.